identity/webapp/pages/account.tsx

```diff
@@ -0,0 +1,290 @@
+import React, { FC, ComponentProps, useState, useEffect } from 'react';
+import Icon from '@weco/common/views/components/Icon/Icon';
+
+import {
+  useUserInfo,
+  withUserInfo,
+} from '@weco/common/views/components/UserInfoContext';
+import { ChangeDetailsModal } from '../src/frontend/MyAccount/ChangeDetailsModal';
+import { PageWrapper } from '../src/frontend/components/PageWrapper';
+import {
+  Container,
+  Title,
+  Header,
+  Intro,
+} from '../src/frontend/components/Layout.style';
+import {
+  SectionHeading,
+  StatusAlert,
+  Wrapper,
+  StyledDl,
+  StyledDd,
+  ProgressBar,
+  ProgressIndicator,
+} from '../src/frontend/MyAccount/MyAccount.style';
+import { Loading } from '../src/frontend/MyAccount/Loading';
+import { ChangeEmail } from '../src/frontend/MyAccount/ChangeEmail';
+import { ChangePassword } from '../src/frontend/MyAccount/ChangePassword';
+import { DeleteAccount } from '../src/frontend/MyAccount/DeleteAccount';
+import { UpdateUserSchema } from '../src/types/schemas/update-user';
+import { useRouter } from 'next/router';
+import WobblyEdge from '@weco/common/views/components/WobblyEdge/WobblyEdge';
+import Layout12 from '@weco/common/views/components/Layout12/Layout12';
+import Layout10 from '@weco/common/views/components/Layout10/Layout10';
+import Space from '@weco/common/views/components/styled/Space';
+import Table from '@weco/common/views/components/Table/Table';
+import { font } from '@weco/common/utils/classnames';
+import { RequestsList } from '@weco/common/model/requesting';
+import { allowedRequests } from '@weco/catalogue/components/ConfirmItemRequest/ConfirmItemRequest';
```

Comment:
These inter-app dependencies will probably cause us headaches further down the line - can we move this into common?

Reply:
Nice catch - I've been trying to think of a way to not allow this to work...